Azure-Samples/Cognitive-Services-Voice-Assistant
# Chschrae/nightfury

## Purpose

Adding cpp client sample with audio players for linux and windows.

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
```

* Test the code

You should be able to build for windows and linux by following the readme

## What to Check
Verify that the following are valid
You should be able to build for windows and linux by following the readme

## Other Information
